src/brb_membership.rs

```diff
@@ -99,11 +134,16 @@ where
     }
 }
 
+/// represents one vote of an Actor
 #[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
 pub struct Vote<A: Ord, S: Ord> {
+    /// generation in which the vote occurs.
```

Comment:
The gen field will be the same as the pending_gen. ie. It's the generation that we are voting for.

Our first generation is 0, but the first votes that will be sent out will be for gen == 1

Reply:
yeah, that's what I had in my head.   not sure if that makes the above comment wrong though, or perhaps just incomplete.    how about:

> /// generation for which the vote occurs.  This will match State::pending_gen

or what would you suggest?

Reply:
yeah that's good :+1: